### DISCUSSION TITLE
Why Can´t install Netbeans IDE 18 , 17 and 8.2 ?

### DISCUSSION THREAD

**Discussion by ezCriss** [apache/netbeans#6272] [Q&A]
Tried to install two versions of netbeans(18 and 8.2 ) and setup windows shows the same error after accepting terms and clicking on next. 
![image](https://github.com/apache/netbeans/assets/138903549/e16f54d7-1b81-4eca-9b5b-5404def0c955)
I tried with jdk 8.0_202 ; jdk 1.17 17.0.8; jdk 18.0.2.1 and changing enviroment variables, but nothing changes.
my S.O arch are x64 bits my windows specs are 
![image](https://github.com/apache/netbeans/assets/138903549/e70781a7-c47b-4f23-96c3-3662ebf4a9c1)

and here is the log file: 
[20230729102112.log](https://github.com/apache/netbeans/files/12207474/20230729102112.log)
Note : I had netbeans installed long before, but this week 23th-27 July I had problems opening projects or adding external libraries (the file chooser wasn't loading, and from what I could find on the web and the details of the error, I only know that it was related to JFileChooser or something like that), so I tried to reinstall it and did not allow me.

**Comment by neilcsmith-net** [comment]
Seems like the installer may also be affected by the JDK issue outlined in #5687 

You could try the community installers, which are not Java based. However you may have to work around the issue in the IDE too, as outlined in that issue.

**Reply by ezCriss** [reply]
JFileChooser related problems(Note of the post) was solved following comments on the issue outlined, thanks, but main problem on the oficial installer is another, in the log(posted before note) file before error says _:" [2023-07-28 11:01:00.342]: NameResolver - to parse Insufficient disk space for extracting the installation data. Additional {1} is required in {0}."_    I understand that it refers to available storage but i have 60gb free. of 200gb on disk.
